### ISSUE TITLE
[source-postgres] Unable to handle uuid arrays

### ISSUE THREAD

**Issue by DamGameHero** [airbytehq/airbyte#29890]
### Connector Name

source-postgres

### Connector Version

3.1.5

### What step the error happened?

During the sync

### Revelant information

Hello!
I'm trying to sync a Postgres instance into BigQuery using CDC.
Example of Postgres table:
```
CREATE TABLE test_uuid_array (id SERIAL PRIMARY KEY, col1 VARCHAR(200), col2 uuid[]);

INSERT INTO public.test_uuid_array(col1,col2) VALUES('record1', '{c32d69e6-bed0-4adf-a03e-eef3b288da35,659102f4-d3b9-401c-ab19-94c9e88eafc6}');
INSERT INTO public.test_uuid_array(col1,col2) VALUES('record2', '{8f12e051-cec3-49db-8b7e-f791720917f0,1a01c716-ec0e-4a6d-8939-65c152631442}');
```
But `uuid` arrays from Postgres appear as `null` in BigQuery (col2 column in the picture below).
![image](https://github.com/airbytehq/airbyte/assets/14892491/e6a04e82-d926-4825-b311-3afd20257e08)

I tested with other version (1.0.43) of the Postgres connector and it worked. It seems to be a regression. 

### Relevant log output

```shell
2023-08-28 08:41:06 WARN i.a.w.i.FieldSelector(lambda$reportMetrics$1):124 - Schema validation errors found for stream public_test_uuid_array. Error messages: [$._ab_cdc_deleted_at: null found, string expected, $.col2: string found, array expected]
```

### Contribute

- [ ] Yes, I want to contribute

**Comment by mrhallak** [comment]
This is also not working for `3.1.7`

**Comment by evantahler** [comment]
This is a rather old issue - is this still occurring?

**Comment by postamar** [comment]
Closing as duplicate of airbytehq/airbyte#34986 , please reopen if I'm wrong.

**Comment by theyueli** [comment]
I just verified this test case using the latest postgres and bigquery connectors. The uuid arrays can be correctly synced to the destination. 

For 3.1.5 PG source connector, I see this issue in a local pg->pg sync. So, the issue has probably already been fixed. 

Closing the ticket.

![Screenshot 2024-05-23 at 3 41 57 PM](https://github.com/airbytehq/airbyte/assets/61070669/23d0492d-786a-4819-b870-b837460544c6)
